oaknational/Oak-Web-Application
# feat(home page): add avo analytics to the visit help center button

## Description

add avo analytics to "visit help center" button

## Issue(s)

Fixes #1098 

## How to test

1. go to home page 
2. Click on  "visit help centre"
3. You should see analytics is Posthog

## Screenshots



How it should now look: the same as before.

## Checklist
